scipy/stats/tests/test_multicomp.py

```diff
@@ -220,43 +252,106 @@ def test_ttest_ind(self, alternative):
             assert_allclose(res.pvalue, ref.pvalue, atol=1e-3)
 
     @pytest.mark.parametrize(
-        'alternative, statistic, pvalue',
+        'alternative, pvalue',
         [
-            ('less', 'low', 0),
-            ('less', 'high', 1),
-            ('greater', 'low', 1),
-            ('greater', 'high', 0),
-            ('two-sided', 'low', 0),
-            ('two-sided', 'high', 0)
+            ('less', [0, 1]),
+            ('greater', [1, 0]),
+            ('two-sided', [0, 0]),
         ]
     )
-    def test_alternatives(self, alternative, statistic, pvalue):
+    def test_alternatives(self, alternative, pvalue):
```

Comment:
The changes to this test make sense. Also, I checked that the numerical results match those produced by `glht`.

<details>

```R
library(multcomp)
options(digits=16)
control <- c(87, 96, 93, 86, 89, 93, 85, 80, 92, 94)
t <- c(13, 7, 19, 7, 19,  9, 18, 13, 16, 19)
w <- c(160, 173, 173, 161, 162, 169, 167, 169, 171, 170)

groups = factor(rep(c("control", "t", "w"), times=c(length(control), length(t), length(w))))
df <- data.frame(response=c(control, t, w), group=groups)
model <- aov(response~group, data = df)
test <- glht(model=model, linfct=mcp(group="Dunnett"), alternative="t")
summary(test)
confint(test)
```

</details>